javascript/builtins/Function.json

```diff
@@ -107,6 +107,38 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "enforces_trusted_types": {
+            "__compat": {
+              "description": "Arguments require `TrustedScript` instance when trusted types are enforced",
```

Comment:
Tested with https://wpt.live/trusted-types/eval-function-constructor-untrusted-arguments-and-applying-default-policy.html

Reply:
```suggestion
              "description": "Requires arguments to be `TrustedScript` instance when trusted types are enforced",
```